ui/src/org/pentaho/di/ui/xul/KettleXulLoader.java

```diff
@@ -55,6 +55,13 @@ public void setIconsSize( int width, int height ) {
     iconHeight = height;
   }
 
+  /**
+   * Get original stream without svg->png transformation.
+   */
+  public InputStream getOriginResourceAsStream( String resource ) {
```

Comment:
@AlexanderBuloichik rename method to match "original" instead of "origin" (probably typo)